mod/lostpass.php

```diff
@@ -64,17 +64,14 @@ function lostpass_post(App $a)
 		Site Location:	%2$s
 		Login Name:	%3$s', $resetlink, DI::baseUrl(), $user['nickname']));
 
-	notification([
-		'type'     => SYSTEM_EMAIL,
-		'language' => $user['language'],
-		'to_name'  => $user['username'],
-		'to_email' => $user['email'],
-		'uid'      => $user['uid'],
-		'subject'  => DI::l10n()->t('Password reset requested at %s', $sitename),
-		'preamble' => $preamble,
-		'body'     => $body
-	]);
-
+	$email = DI::emailer()
+	           ->newSystemMail((!empty($user['language'])) ? DI::l10n()->withLang($user['language']) : DI::l10n())
```

Comment:
Code standards: Please use only tabs for indenting.

Reply:
Hmm .. The nice thing in this case is that the method-chains are aligned at the called method. If we use tabs, most of the time they are not I guess .. Do you know the right setting in PHPStorm for aligning new-lines with tabs? I don't find it and it's dangerous to manually override these settings (everytime I re-format the code, I've to manually change it as well..)

Reply:
Not off the top of my head, but searching for “Tabs” in settings should show all relevant keys.

Reply:
I manually changed these lines for now (I found all PHP Tab settings but nothing which changes this behavior :( )

Are you sure this is better? I found the other type better readable

Reply:
OK nevermind .. I think I can live with it too since the "meaning" starts with the call of `newSystemEmail()` and stay in a line.. OK I change the rest